include/votca/xtp/bse.h

```diff
@@ -146,13 +146,24 @@ class BSE {
   void Solve_singlets_TDA();
   void Solve_singlets_BTDA();
 
+  void Solve_triplets_TDA();
+  void Solve_triplets_BTDA();
+
+  void PrintWeight(int i, int i_bse, QMStateType type);
+
   template <typename BSE_OPERATOR>
   void configureBSEOperator(BSE_OPERATOR& H);
 
   template <typename BSE_OPERATOR>
   void solve_hermitian(BSE_OPERATOR& H, Eigen::VectorXd& eigenvalues,
                        Eigen::MatrixXd& coefficients);
 
+  template <typename BSE_OPERATOR_ApB, typename BSE_OPERATOR_AmB>
+  void Solve_antihermitian(BSE_OPERATOR_ApB& apb, BSE_OPERATOR_AmB& amb,
```

Comment:
just to be annoying : `solve_` instead of `Solve_` :)

Reply:
true, I merged before I saw the comments, I ll fix it in the main version.